### ISSUE TITLE
Download Analytics button on CurseForge has disappeared

### ISSUE THREAD

**Issue by Requios** [antoinegag/curseforge-analytics-viewer#21]
The instruction "Head to the CurseForge project dashboard and click on the Download button in the Analytics column." doesn't seem to be working anymore, there is no Download button for any sort of analytics. Is there a different place I can find it, or has it been removed entirely?

**Comment by antoinegag** [comment]
Yeah... Overwolf (who now owns CurseForge) have completely removed the option to download your analytics. 

They said they would add a tool to see your analytics directly in the site but no idea when it'll be back. Sorry :/
